src/generic/grid.cpp

```diff
@@ -10332,4 +10327,44 @@ wxGridCellEditor* wxGridTypeRegistry::GetEditor(int index)
     return editor;
 }
 
+wxRect GetGridCheckBoxRect(const wxSize& checkBoxSize,
+                           const wxRect& cellRect,
+                           int hAlign, int WXUNUSED(vAlign))
+{
+    // TODO: support vAlign
+
+    wxRect checkBoxRect;
+    checkBoxRect.SetY(cellRect.y + cellRect.height / 2 - checkBoxSize.y / 2);
+
+     wxCoord minSize = wxMin(cellRect.width, cellRect.height);
```

Comment:
```suggestion
    wxCoord minSize = wxMin(cellRect.width, cellRect.height);
```
Indentation seems to be wrong.